src/sentry/static/sentry/app/components/circleIndicator.tsx

```diff
@@ -2,15 +2,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 
-const getBackgroundColor = p => {
+type Props = {
```

Comment:
You should merge the the div attribute props to, since we spread props onto the div element.

But if you make it a styled component, emotion will add those props for you